invenio_previewer/__init__.py

```diff
@@ -234,78 +246,61 @@
 
```

Comment:
What about changing the beginning of each bullet point focusing first in the use case?

- Per file: ...
- Globally: ...

You can rephrase OFC :)